utilities/manual/usermanual_en.html

```diff
@@ -1487,37 +1487,37 @@ <h2 id="SHORTCUTS">4.13 Keyboard shortcuts</h2>
 	<li>Find : Ctrl+F</li>
 	<li>Find next : F3 / Ctrl+G</li>
 	<li>Find prev : Shift+F3 / Ctrl+Shift+G</li>
-	<li>Replace : CTrl+R</li>
+	<li>Replace : Ctrl+R</li>
 	<li>Go to line : Ctrl+G</li>
-	<li>Go to previous change: Ctrl+H</li>
-	<li>Go to to next change: Ctrl+Shift+H</li>
-	<li>Go to Bookmark 0..9: Ctrl+0..9</li>
-	<li>Set Bookmark 0..9: Ctrl+Shift+0..9</li>
-	<li>Set Unnamed Bookmark: Ctrl+Shift+B</li>
-	<li>Next Marker: Ctrl+Down</li>
-	<li>Previous Marker: Ctrl+Up</li>
+	<li>Go to previous change : Ctrl+H</li>
+	<li>Go to to next change : Ctrl+Shift+H</li>
+	<li>Go to Bookmark 0..9 : Ctrl+1..9</li>		<!-- 04/26/2022 it is assumed that Ctrl+0 would make problems, s. #2275 -->
+	<li>Toggle Bookmark 0..9 : Ctrl+Shift+1..9</li>		<!-- 04/26/2022 it is assumed that Ctrl+Shift+0 would make problems, s. #2275 -->
```

Comment:
ctrl+shift+0 exists, only ctrl+0 is not used

Reply:
You are right, another fault of the microsoft sandbox: Both shortcuts do not work there (I asssume that the reason is not Version TeXstudio 4.2.4 (git 348e51fe6) I currently use in the sandbox).
I understand that txs doesn't use ctrl+0 for anything. So we could add it, because without jumping to the bm, bm 0 is not so usefull. Or do you want to keep the state as is?